tests/test_schema.py

```diff
@@ -45,3 +45,80 @@ def test_relative_path_not_match_schema(data, schema):
     data["rhui_alias"][0]["dest"] = "dest-path"
     with pytest.raises(ValidationError):
         jsonschema.validate(data, schema)
+
+
+def test_empty_list(data, schema):
+    """Verify that YAML returns empty list."""
+    assert data["signing_keys_mappings"]["platform"]["middleware"]["ga_keys"] == []
+
+
+def test_invalid_arches(data, schema):
```

Comment:
I only added tests to validate regexes. I wasn't sure that it was necessary to include tests that verify that the json schema won't allow additional properties that don't match the specified type, or pattern (if the type is a string), etc.